test/Microsoft.Extensions.ProjectModel.Tests/project.json

```diff
@@ -7,10 +7,11 @@
     }
   },
   "dependencies": {
-    "NuGet.Frameworks": "3.5.0-rc1-final",
+    "NuGet.ProjectModel": "3.6.0-*",
```

Comment:
@prafullbhosale @NTaylorMullen just ran into the problem that @davkean predicted. Project evaluation is through MSBuild APIs is going to be really hard to do right because as soon as the SDK changes their references, our project evaluation fails.  We should try to get off using MSBuild API directly as soon as possible.